src/Report/Xml/Project.php

```diff
@@ -63,19 +63,12 @@ public function buildInformation(): BuildInformation
 
     public function tests(): Tests
     {
-        $testsNode = $this->contextNode()->getElementsByTagNameNS(
```

Comment:
we call this method on a document, which at this point of time cannot already contain a `<tests>` element.